roles/setup_pgd/defaults/main.yml

```diff
@@ -113,3 +113,12 @@ supported_pg_version:
 
 supported_pgd_version:
   - 5
+
+# Other PG Users
+enterprisedb_user: "enterprisedb"
+
+# PGD CAMO
+# PGD Cluster Node To remove for CAMO
+# PGD CAMO configuration requires a maximum of two nodes
+pgd_node_names_to_remove: ["edb_primary3"]
```

Comment:
I still don't love this idea of removing the PGD node, if a max of two nodes is required for CAMO, whats the point of a HA cluster? Upon further looking through the role and PGD documentation, I ran into a few concerning things that I will bring up on Monday.

Reply:
I discussed this briefly with Doug, but we maybe need another sanity check.  If CAMO simply does not work when there is more than 2 data nodes, then we need to update our test to use 2 nodes. 

Having said that, I don't know if the edb-ansible role needs any changes just to build 2 nodes.  Or whether that is a significant development effort.